### ISSUE TITLE
Should preserve metadata

### ISSUE THREAD

**Issue by tbm** [beancount/beancount2ledger#7]
Metadata is currently lost in conversion:

```
2020-07-17 * "Test"
  string: "foo"
  year: 2020
  date: 2020-07-19
  Assets:Test     10.00 EUR
  Assets:Test    -10.00 EUR
```

Result:

```
2020-07-17 * Test
  Assets:Test                                                      10.00 EUR
  Assets:Test                                                      -10.00 EUR
```

**Comment by tbm** [comment]
This is a duplicate of issue #3.

**Comment by tbm** [comment]
Fixed in 381fc1a98a33d6ffd7f16
